source/Calamari.Tests/Calamari.Tests.csproj

```diff
@@ -34,6 +34,10 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\MarkdownSharp.1.13.0.0\lib\35\MarkdownSharp.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Web.Administration, Version=7.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <Private>True</Private>
+      <HintPath>..\packages\Microsoft.Web.Administration.7.0.0.0\lib\net20\Microsoft.Web.Administration.dll</HintPath>
```

Comment:
No, including this reference in the test project ensures that the dll is copied over so that the tests run. As the assembly is referenced by classes loaded via reflection it doesn't get brought into the bin directory in Calamari.Tests